src/test/java/com/google/javascript/clutz/inherit_visibility.js

```diff
@@ -0,0 +1,39 @@
+goog.module('inherit_visibility');
+
+/** @constructor */
+function Parent() {}
+
+Parent.prototype.publicImplicit = function() {};
+/** @public */
+Parent.prototype.publicExplicit = function() {};
+/** @protected */
+Parent.prototype.protected = function() {};
+/** @protected */
+Parent.prototype.protectedNoJSDoc = function() {};
+
+/** @constructor @extends {Parent} */
+function Child() {}
+
+/** @override */
+Child.prototype.publicImplicit = function() {};
+/** @override */
+Child.prototype.publicExplicit = function() {};
+/** @override */
+Child.prototype.protected = function() {};
+Child.prototype.protectedNoJSDoc = function() {};
+
+/** @constructor @extends {Child} */
+function Child2() {}
```

Comment:
GrandChild :) Otherwise, one would read it as another class extending Parent.

Reply:
fixed!